airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/JobCreationAndStatusUpdateActivity.java

```diff
@@ -131,6 +131,7 @@ class JobFailureInput {
 
     private long jobId;
     private String reason;
+    private AttemptFailureSummary failureSummary;
```

Comment:
You need to add a new method instead of adding one (related to my previous comment).